microsoft-edge/toc.yml

```diff
@@ -1338,6 +1338,31 @@
                   href: /microsoft-edge/webview2/reference/win32/index
                   displayName: functions, methods
 
+            - name: JavaScript APIs in WebView2
```

Comment:
```suggestion
            - name: JavaScript
```

Reply:
The above changes makes it parallel with other nodes in the ref doc node.

Reply:
We are trying to figure out how to distinguish this set of APIs from the others actually. The others are the WebView2 APIs you use to create and use the WebView2 and are roughly the same thing but in .NET/COM/WinRT. These JavaScript APIs are APIs that WebView2 exposes to the JavaScript running in the WebView2 and are mostly about communication between the JavaScript and the app hosting the WebView2 (although not necessarily limited to that going forward).

That's why its called JavaScript APIs *in* WebView2. But maybe there's other things we could call this or do to make it clearer?